Doc/library/sqlite3.rst

```diff
@@ -335,6 +343,9 @@ Module functions
    .. versionadded:: 3.10
       The ``sqlite3.connect/handle`` auditing event.
 
+   .. versionadded:: 3.12
+      The *autocommit* attribute.
+
 .. function:: complete_statement(statement)
```

Comment:
`isolation_level` is not deprecated yet. I'll mention this in the issue as well, but I'll write my thoughts out here just for the record: I initially intended to change the default `autocommit` value in 3.14 (as the PR docs currently say), but that implies introducing an already deprecated constant, `sqlite3.DEPRECATED_TRANSACTION_CONTROL`, which is kinda a weird thing to do, and I should probably leave more time for users to adapt, _and_ it implies deprecating `isolation_level` now in 3.12, because having a different `autocommit` default _and_ keeping the current default for `isolation_level` makes no sense at all. So, a better plan is probably to introduce the `autocommit` attribute now (as we are doing), chose a slightly more friendly name for the deprecated-transaction-control-constant (`COMPAT_TRANSACTION_CONTROL`?), and wait until 3.13 or preferrably 3.14 before introducing the deprecations.

Reply:
IOW, we introduce `autocommit` as the new _recommended way of doing things_, but we still keep the current defaults; no-one needs to change their code... yet.

Reply:
I wrote a detailed reply to this yesterday, but to my extreme frustration, GitHub seems to have somehow lost it.

In any case, I believe the gist of what I said was:

* Regardless of what the plan is (whether that's deprecate now, deprecate later or never deprecate), it should be clearly communicated to users in the documentation. This both reduces fear, uncertainty and doubt surrounding the deprecation, and aids and encourages users to plan for, raise feedback about and implement the migration smoothly and in plenty of time, making things easier for everyone.
* My suggestion was `LEGACY_TRANSACTION_CONTROL` for the name. This is clearer and more specific than `COMPAT` transaction control (compat with what? My first thought would be compatibility with other database types or backends—when in fact it is the PEP 249 behavior that is perhaps more "compatible" in that sense—not retaining legacy behavior from an older Python version,), and also helps gently guide users toward the more modern PEP 249 behavior, without deprecating the old behavior (yet).
* Even if `isolation_level`/etc. isn't deprecated, the new `autocommit` way of doing things should be at least mentioned in the `isolation_level` parameter and attribute entries, to help make users aware of it and guide them toward it as now rather than only right when its deprecated.

In any case, now that you've updated the PR, I'll do another full review, and if any of the above feedback is still relevant, I'll make specific comments and suggestions accordingly.